app/src/main/kotlin/com/hippo/ehviewer/ui/reader/GalleryPager.kt

```diff
@@ -61,8 +61,7 @@ fun GalleryPager(
         } else {
             onNavigationModeChange()
```

Comment:
When the reading mode changes but the navigation mode remains the same, this won't be called and therefore the navigation overlay won't show.

Reply:
Actually, maybe it's fine.
Navigation mode not changed = no navigation overlay, seems intuitive.